docs/concepts/gardenlet.md

```diff
@@ -231,6 +235,12 @@ For example, if you set it to `48`, then the `BackupEntry`s for deleted `Shoot`s
 Additionally, you can limit the [shoot purposes](../usage/shoot_purposes.md) for which this applies by setting `.controllers.backupEntry.deletionGracePeriodShootPurposes[]`.
 For example, if you set it to `[production]` then only the `BackupEntry`s for `Shoot`s with `.spec.purpose=production` will be deleted after the configured grace period. All others will be deleted immediately after the `Shoot` deletion.
 
+#### "Migration" Reconciler
+
+This Reconciler is added to the manager only if the [`ForceRestore`](../deployment/feature_gates.md#list-of-feature-gates) feature gate is enabled in the gardenlet and if the seed has owner checks enabled (ie; spec.setttings.ownerChecks.enabled=true). 
+This reconciler checks if the source seed also has owner checks enabled and if it is not, or the BackupEntry is being deleted, it sets the `status.MigrationStartTime` to nil.
+Or else, if the `BackupEntry` has force restore annotation, ie; `shoot.gardener.cloud/force-restore=true` or the Grace period for migration has elapsed, which is set in the `BackupEntryMigrationControllerConfiguration` in the gardenlet configuration, or if the `lastOperation` in the BackupEntry was `Migrate` and if `LastOperationStaleDuration` in the controller configuration has passed since the  `lastUpdateTime` of the `BackupEntry`, it updates the status to force restoration.
```

Comment:
```suggestion
Otherwise, if the `BackupEntry` is annotated with `shoot.gardener.cloud/force-restore=true` or if the grace period for migration has elapsed (which is set in the `BackupEntryMigrationControllerConfiguration` in the gardenlet's component configuration), or if the `lastOperation` in the BackupEntry was `Migrate` and if `LastOperationStaleDuration` in the controller configuration has passed since the  `lastUpdateTime` of the `BackupEntry`, it updates the status to force restoration.
```

Generally, this sentence is unreadable. Please simplify this drastically so that it is valueable.